spring-cloud-common-security-config/spring-cloud-common-security-config-core/pom.xml

```diff
@@ -29,5 +29,10 @@
 			<artifactId>javax.servlet-api</artifactId>
 			<scope>provided</scope>
 		</dependency>
+		<dependency>
```

Comment:
If this is required, we could have just added it where it was before at the 2 roots (scdf-parent and scdf-build-dependencies). I am guessing this will come out in the next phase of the PR though.

Reply:
Removing all junit 4 at the end will be easy. If everything builds it is easier to review.